docs/maintainers/maintainers-guide.md

```diff
@@ -0,0 +1,44 @@
+---
+id: maintainers-guide
+title: "@open-sauced/maintainers-guide"
+sidebar_label: "@open-sauced/maintainers-guide"
+keywords:
+  - "@open-sauced/maintainers-guide"
+---
+
+Welcome to OpenSauced, Maintainers Guide for contributors who are interested in being part of the maintainer team for OpenSauced repositories. 
```

Comment:
[proofreading] This is a great start @BekahHW! :) Just omit the parts I suggested below so the paragraph can be clearer. 
```suggestion
Welcome to OpenSauced! This guide is for contributors who are interested in being part of the maintainer team!  
```